test/DocumentFormat.OpenXml.Packaging.Tests/PartUriHelperTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.IO;
+using System.IO.Packaging;
+using Xunit;
+
+namespace DocumentFormat.OpenXml.Packaging.Tests
+{
+    public class PartUriHelperTests
+    {
+        [InlineData("unknown", "/", "/target", "/target")]
+        [InlineData("unknown", "/something", "/target", "/target")]
+        [InlineData("application/vnd.openxmlformats-officedocument.wordprocessingml.footer+xml", "/target", "target", "/target")]
+        [Theory]
+        public void GetUniquePartUriTest(string contentType, string parentUri, string targetUri, string expected)
+        {
+            var helper = new PartUriHelper();
+            var unique = helper.GetUniquePartUri(contentType, new Uri(parentUri, UriKind.Relative), new Uri(targetUri, UriKind.Relative));
+
+            Assert.Equal(new Uri(expected, UriKind.Relative), unique);
+        }
+
+        [InlineData("unknown", "/", "/target", "/target2")]
+        [InlineData("unknown", "/something", "/target", "/target2")]
+        [InlineData("application/vnd.openxmlformats-officedocument.wordprocessingml.footer+xml", "/target", "target", "/target1")]
```

Comment:
@Josh-Curry You added support for some namespaces to append the 1, but it appears that this is only the case for one of the overloads of `PartUriHelper.GetUniqueParUri`. Is this expected, or should it be added for all?